RIOT-OS/RIOT
# driver/cc110x: Fixed indent/alignment in doc

### Contribution description

The Doxygen documentation blocks have not been correctly aligned. This commit fixes this.

### Testing procedure

Check that this PR indeed only touches comments. (And one parameter is renamed, as it was to long.) If no code has been touched, this PR will not break anything.

and/or

Build some network application, e.g. `examples/gnrc_networking` with the `cc110x` module used (e.g. by building for the `msba2` or the `msbiot`). The binary before and after applying this PR should be identical, as no code was changed.

### Issues/PRs references

None
